airgun/widgets.py

```diff
@@ -1118,6 +1118,31 @@ class EditableEntryCheckbox(EditableEntry):
     edit_field = Checkbox(locator=".//input[@type='checkbox']")
 
 
+class EditableEntryMultiCheckbox(EditableEntry):
+    """Should be used in case :class:`EditableEntry` widget represented not by
+    a field, but by a set of checkboxes.
+    """
+
+    def fill(self, value):
+        """Check or unxeck one of the checkboxes
+
+        :param value: string with specification of fields' values
+            Example: value={'details.addons': {'Test addon 1': True, 'Test addon 2': False}}
+        """
+
+        if self.edit_button.is_displayed:
```

Comment:
that if doesn't make sense since you cannot fill without clicking the edit button. In case that button is not displayed - we get the exception, which is expected behavior.
Initial code was generic, here you know exact scenario for your widget